Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -143,6 +143,40 @@ protected theorem induction_on {motive : M ⊗[R] N → Prop} (z : M ⊗[R] N)
       exact add _ _ (tmul ..) ih
 #align tensor_product.induction_on TensorProduct.induction_on
 
+/-- Lift a map that is additive in both arguments, such that scalar multiplication in either
+argument is equivalent, to the tensor product.
+
+Note that strictly the first action should be a right-action by `R`, but for now `R` is commutative
+so it doesn't matter. -/
+-- TODO: use this to implement `lift` and `SMul.aux`. For now we do not do this as it causes
+-- performance issues elsewhere.
+def liftAddHom (f : M →+ N →+ P)
+    (hf : ∀ (r : R) (m : M) (n : N), f (r • m) n = f m (r • n)) :
+    M ⊗[R] N →+ P :=
+  (addConGen (TensorProduct.Eqv R M N)).lift (FreeAddMonoid.lift (fun mn : M × N => f mn.1 mn.2)) <|
+      AddCon.addConGen_le fun x y hxy =>
```

Comment:
Is indent four spaces right here?

Reply:
I think this is copied almost verbatim from `SMul.aux` below (deduplicating is left for another PR due to performance complications), so the indent is mathport-approved if nothing else.

Reply:
I take it back, you were right!